scripts/iso-manager.sh

```diff
@@ -28,11 +28,15 @@ ARCHIVE_DEPRECATED_OPTION=
 
 while (( "$#" )); do
   case "$1" in
-    -a|--archive|--add-archive)
+    --archive)
       ARCHIVE_DEPRECATED_OPTION="true"
```

Comment:
Or maybe just

```suggestion
      [ "$1" = "--archive" ] && ARCHIVE_DEPRECATED_OPTION="true"
```

?

Reply:
Yes, ok ok

Reply:
Fixed